packages/desktop-client/src/components/reports/graphs/LineGraph.tsx

```diff
@@ -54,12 +53,9 @@ const CustomTooltip = ({ active, payload }: CustomTooltipProps) => {
           </div>
           <div style={{ lineHeight: 1.5 }}>
             <PrivacyFilter>
-              <AlignedText left="Assets:" right={payload[0].payload.assets} />
```

Comment:
Didn't see this used anywhere, but I could combine them if there were plans for it.